package.json

```diff
@@ -2,42 +2,20 @@
   "name": "smi-spec.io",
   "version": "1.0.0",
   "description": "site for service mesh interface",
-  "main": "gulpfile.js",
-  "dependencies": {
-    "del": "^5.1.0",
-    "font-awesome": "^4.7.0",
-    "foundation-sites": "~5.5.3",
-    "gulp": "^4.0.0",
-    "gulp-autoprefixer": "^7.0.1",
-    "gulp-cache": "^1.0.2",
-    "gulp-clean-css": "^4.0.0",
-    "gulp-cssnano": "^2.1.3",
-    "gulp-git": "^2.9.0",
-    "gulp-imagemin": "^7.0.0",
-    "gulp-livereload": "^4.0.0",
-    "gulp-rename": "^2.0.0",
-    "gulp-sass": "^4.0.2",
-    "gulp-sourcemaps": "^2.6.4",
-    "node-sass": "^4.11.0",
-    "streamqueue": "^1.1.2"
-  },
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "author": "deislabs",
+  "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/deislabs/smi-spec.io.git"
```

Comment:
isn't the URL https://github.com/servicemeshinterface/smi-spec.io now?

Reply:
Thanks Bridget, updated this now that the repos have moved around